### DISCUSSION TITLE
Retrieving "Organism|R*T" parameter using ospsuite R package

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1730] [Questions & Problems] [answered]
I am trying to retrieve the "Organism|R\*T" parameter but I have added other parameters to the organism that are now being picked up. I can see that the issue comes from the "\*" within the parameter name. How does one specify a literal "*"? I tried using regex expression, but I have not managed to get it to work. I have attached a test example below.

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/57540473/6ae53729-cdd2-4ccd-9a21-48e83a72560d)

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/57540473/d7f73675-f8a6-4ec0-b875-d041ec54d2b3)

[test.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/14615889/test.zip)

**Comment by msevestre** [comment, marked as answer]
@moPellowe 
You can't escape this. * Is a special character that we use as a wildcard. We did not think about the fact that `*` would be part of the name of parameters. In general, * should not be allowed as a parameter name. @Yuri05. Do you agree?

**Reply by Yuri05** [reply]
agree
